sceptre/strides/config/prod/github-oidc-sage-bionetworks-admin.yaml

```diff
@@ -0,0 +1,16 @@
+template:
+  type: http
+  url: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.7.6/templates/IAM/github-oidc-provider.j2
+stack_name: github-oidc-sage-bionetworks-admin
```

Comment:
why such a general name?  is the purpose to use this OIDC for multiple GH repos?

Reply:
That was the idea, this role can be assumed by any repo in the SageBio org, and the admin was meant to indicate the permissions assumed. Same for the renaming below. I'm fine with using the simpler naming convention below (i.e. 'admin' is assumed unless otherwise specified) but I think we need to move away from naming the roles after a repo since we now support multiple repos... Or am I missing the point?